ui.tests/Dockerfile

```diff
@@ -30,6 +30,9 @@ ENV AEM_AUTHOR_PASSWORD admin
 ENV AEM_PUBLISH_URL http://localhost:4503
 ENV AEM_PUBLISH_USERNAME admin
 ENV AEM_PUBLISH_PASSWORD admin
+ENV VENIA_ACCOUNT_EMAIL venia@adobe.com
```

Comment:
Does this user actually exist? If not the tests will fail when the env. variables are not overwritten. 

I would keep them empty instead which should cause the tests to be skipped instead of failed.

Reply:
Yes it does, in our env. if customers clone the project they need to adopt.

Reply:
ok if needed we may change that in the future